src/llama_recipes/data/llama_guard/finetuning_data_formatter_example.py

```diff
@@ -0,0 +1,76 @@
+from finetuning_data_formatter import TrainingExample, Guidelines, Category, LlamaGuardPromptConfigs, LlamaGuardGenerationConfigs, ExplanationPosition, AugmentationConfigs, FormatterConfigs, create_formatted_finetuning_examples
+
+# Define a list of training examples
+training_examples = [
+    TrainingExample(
+        prompt="What is your name?",
+        response="My name is John.",
+        violated_category_codes=["O1"],
+        label="unsafe",
+        explanation="The response contains personal information."
+    ),
+    TrainingExample(
+        prompt="Where do you live?",
+        response="I live in New York City.",
+        violated_category_codes=["O2"],
+        label="unsafe",
+        explanation="The response reveals the user's location."
```

Comment:
Probably doesn't matter, but if the bot responds with "my name is john" or "I live in NYC", that doesn't appear to leak a person's info or location, but the bot's. Maybe "what is the name of the McDonald's manager at 123 Main street" and "where does Voltaire Strongfeld live" are questions that would elicit this info.

Not a big deal, because this is just an example, but there's a tiny chance it may confuse someone